### ISSUE TITLE
Broken Rx() arg outputs=[]

### ISSUE THREAD

**Issue by cstarkjp** [gprMax/gprMax#431]
**Cannot pass list or tuple of output fields to `Rx()`**

`Rx()` argument `outputs` is meant to be an iterable, e.g., `outputs=["Ex", "Ey", "Ez", "Ix", "Iy", "Iz",]`, but this fails in v4 (and also I believe in v3) because `cmds_multiuse.py` (line 874) does this:  `outputs = [self.kwargs["outputs"]]` making the parameter into a list of lists. Worse, the sort on line 881 requires this iterable to be mutable, disallowing the passing of a tuple.

If a single literal only is passed, e.g., `outputs="Ex"`, a list containing this literal is generated, so the parameter is accepted. 

In other words, **it is currently only possible to specify a single field component, not several;** and a tuple is not allowed.

Fix: (to line 874) `outputs = list(self.kwargs["outputs"])` 

This allows the `outputs.sort()` on line 881 to work even if a tuple rather than a list is passed.

**Comment by craig-warren** [comment]
@cstarkjp thanks I've implemented your fix https://github.com/gprMax/gprMax/commit/7f4ccc8b28b94aeafb51b0767dff9117ad96baa3

**Comment by craig-warren** [comment]
This should now be fixed by https://github.com/gprMax/gprMax/commit/9f4bb79f54d47f040b49f88cbd9ad6630bfb2d4f We are now enforcing the use of a list when specifying even a single output component, e.g. `outputs = ["Ey"]` or `outputs = ["Ex", "Ey", "Ez"]`
